docs/use-cases/ratelimit/README.md

```diff
@@ -0,0 +1,219 @@
+<!-- omit in toc -->
+# Ratelimit
+
+The purpose of this use case is to deploy a sample application published via AWS ELB, that will be ratelimited at infrastructure level, thanks to the use the envoyproxy sidecar container injected in the app pod by marin3r, that will be in charge of contacting to a third party ratelimit service, that will authorize the request (or not) if it is within the permitted  limits.
```

Comment:
I wouldn't use the word `authorize` in this context as it can be confused with authorization features in envoy. I would go with `allow` for example.

```suggestion
The purpose of this use case is to deploy a sample application published via AWS ELB, that will be ratelimited at infrastructure level, thanks to the use the envoyproxy sidecar container injected in the app pod by marin3r, that will be in charge of contacting to a third party ratelimit service, that will allow the request (or not) if it is within the permitted  limits.
```